### ISSUE TITLE
Loss is not reducing

### ISSUE THREAD

**Issue by renesass** [mperezcarrasco/PyTorch-DAGMM#1]
Hi Manuel,

thank you for providing such a nice implementation of DAGMM for PyTorch!
While I tried out your solution, I found that the loss is not reducing and rather alternating. The original implementation in TensorFlow, however, does not have this problem.
Do you may know where this is coming from?

Best,
René
